gem2rpm/foreman_plugin.spec.erb

```diff
@@ -141,6 +142,10 @@ mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
+<% if has_cronjob -%>
+mkdir -p %{buildroot}%{?scl:%_root_sysconfdir}%{!?scl:%_sysconfdir}/cron.d/
+mv %{buildroot}%{gem_instdir}/extra/*cron* %{buildroot}%{?scl:%_root_sysconfdir}%{!?scl:%_sysconfdir}/cron.d/%{gem_name}
```

Comment:
I think this breaks when there's multiple cron files. Probably a bad idea regardless and maybe something we don't want to support.

Reply:
Yes, it will break. Any ideas for a safeguard? Or is simply "out of support" ok?

Reply:
Given how few of these we have it's probably safe enough to let it fail and let the packager deal with the build failure.